eventsourcing/domain/services/eventstore.py

```diff
@@ -91,13 +90,14 @@ def get_entity_version(self, stored_entity_id, version):
 
 
 class AbstractStoredEventRepository(six.with_metaclass(ABCMeta)):
-
-    def __init__(self, always_check_expected_version=False, always_write_entity_version=False):
+    def __init__(self, always_check_expected_version=False, always_write_entity_version=False,
+                 stored_event_class=Transcoder.StoredEvent):
```

Comment:
Thanks for that! Actually I've also been making the changes we discussed, so I don't know if there is any overlap, but it might be worth checking out #29 "Refactor the transcoder a bit more" :-)